tests/include/igl/quadprog.cpp

```diff
@@ -0,0 +1,137 @@
+#include <test_common.h>
+#include <igl/quadprog.h>
+#include <igl/EPS.h>
+
+
+TEST_CASE("quadprog: box3", "[igl]" )
+{
+  {
+    Eigen::Matrix3d H = (Eigen::Matrix3d(3,3)<<
+        0.240548455386281,   0.237314308102107,  0.0436993831501944,
+        0.237314308102107,   0.326254049041854,-0.00021896091952631,
+        0.0436993831501944,-0.00021896091952631,   0.171175681280756
+        ).finished();
+    Eigen::Vector3d f = (Eigen::Vector3d(3,1)<<
+        0.222182612270718,
+        0.503254616893693,
+        -0.184619987497072
+        ).finished();
+    Eigen::Vector3d lb = (Eigen::Vector3d(3,1)<<
+        -1,
+        -1,
+        -1
+        ).finished();
+    Eigen::Vector3d ub = (Eigen::Vector3d(3,1)<<
+        1,
+        1,
+        1
+        ).finished(); 
+	// Windows needs template args spelled out
+    Eigen::Vector3d x = igl::quadprog<double,3>(H,f,lb,ub);
```

Comment:
So this particular issue seems similar to this [stack overflow issue](https://stackoverflow.com/questions/58520935/avoiding-ambiguous-parameters-in-templated-eigen-functions-with-msvs). The suggestion  would be to add more template arguments to match the `Eigen::Matrix` type. Looks like a MSVC bug for sure...

Reply:
This was such a nightmare to debug. At least I figured out how to compile libigl on windows in the process.